setup.py

```diff
@@ -59,6 +62,14 @@
 
 CMDCLASS = versioneer.get_cmdclass()
 
+from Cython.Build import cythonize
```

Comment:
Recent setuptools (>18.0) can detect if Cython is installed, so no need to import Cython. This is the line that makes it work (in sourmash): https://github.com/dib-lab/sourmash/blob/9b9f7f467b9e99a4bba2e73869058da6021a0835/setup.py#L67